protocol/simplified-guardian.md

```diff
@@ -0,0 +1,127 @@
+after some period of time. Given the potential security concerns with allowing a minority of the
+Security Council to unpause, we prefer option (2).
+
+We therefore propose that the pause would expire after some period of time to be defined in the
+specification for these changes (on the order of several months). Once expired, the pause would
+have a cooldown period (to be specified later) to prevent anyone with access to the pause from
+repeatedly pausing as soon as the previous pause expires. A majority of the Security Council would
+be able to reset the expiry clock or explicitly unpause the system. An explicit unpause action
+(rather than an expiration) would  similarly reset the cooldown such that the pause can again be
+executed immediately.
+
+### Cluster-wide Pause
+
+Because the pause mechanism would become the main mechanism available to the Optimism Foundation,
+the pause mechanism would also become the basis for an updated incident response process. However,
+the Superchain-wide Pause is far too impactful to be the only available option during an incident.
+Given that we suspect many incidents will take the form of *single* impacted chains or chain
+clusters, it may be valuable to implement a Cluster-wide Pause on top of the Superchain-wide Pause.
+A chain would be paused if *either* the Superchain-wide Pause is active or its Cluster-wide Pause
+is active.
+
+## Considerations
+
+### Cluser-wide Pause Complexity
+
+It should be noted that the Cluster-wide Pause is likely the most significant source of complexity
+in this proposal. We have a number of different options for how to implement this. Here's a
+non-exhaustive list:
+
+- Cluster ID inside of `SystemConfig` with Cluster-wide Pause held inside of `SuperchainConfig`.
+- New `ClusterConfig` contract.
+- Pause within `SystemConfig` and bundle pauses for an entire cluster into single transaction.
+- Only use Superchain-wide pause.
+
+Given that for the foreseeable future we expect to either (1) pause individual chains or (2) pause
+the primary interop cluster and (3) that an issue impacting the entire primary interop cluster
+probably requires a Superchain-wide Pause anyway, it may just be simpler to implement an additional
+pause in the `SystemConfig` contract instead.
+
+### Pause Expiry Security Implications
+
+Pause Expiry has non-trivial security implications in that a bug in the proof system alongside a 
+malicious blocking minority of the Security Council could be catastrophic. L2Beat explciitly argues
+that this security model is better than the model in which a malicious Foundation and a blocking
+minority of the Security Model is catastrophic. Since this is fundamental to the updated Stage 1
+definition, we are required to accept this conclusion. We may want to take this as motivation to
+implement some other proof system as a backup to the primary proof system.
+
+### Permissioned Proof System
```

Comment:
Need to do product research on the pause mechanism itself

Reply:
Reference from @ControlCplusControlV: https://docs.lido.fi/contracts/gate-seal/

Reply:
Did we ever do this research?